src/Agent/NewRelic/Profiler/MethodRewriter/FunctionManipulator.h

```diff
@@ -269,11 +263,11 @@ namespace NewRelic { namespace Profiler { namespace MethodRewriter
 #ifdef DEBUG
         void WriteLineToConsole(xstring_t message)
         {
-            _instructions->AppendString(message);
             // System.Console is not available in System.Private.CoreLib it is defined in System.Console which is not already a reference
             // in all assemblies. For now we can just allow this DEBUG build only writing in .net framework apps.
             if (!_isCoreClr)
             {
+                _instructions->AppendString(message);
```

Comment:
This change fixes a bug introduced in a previous commit that breaks the reflection call strategy by introducing by IL.